### ISSUE TITLE
you fuckin' druggo

### ISSUE THREAD

**Issue by TKW905** [ValveSoftware/Source-1-Games#4300]

**Comment by concernedtradermain** [comment]
You are the epitome of that one kid in school who reminds the teacher at the very end of the class that they forgot to collect the homework.

**Comment by spaghetties** [comment]
true lol ^^

**Comment by TF2CutContentWiki** [comment]
❤️

**Comment by MandM99** [comment]
The people attempting to get these bugs fixed are mostly doing this to laugh at people losing their item investments. It damages the item economy and will hurt not only the community, but the game's income.

@kisak-valve, we ask you to reconsider since hundreds of people, including ones who are buying items off the SCM, had their inventory values demolished in the process after the update yesterday.

**Comment by rabscootle** [comment]
> The people attempting to get these bugs fixed are mostly doing this to laugh at people losing their item investments. It damages the item economy and will hurt not only the community, but the game's income.
> 
> @kisak-valve, we ask you to reconsider the fix since hundreds of people, including ones who are buying items off the SCM, had their inventory values demolished in the process after the update yesterday.

The only issue with the quickswitch UI getting fixed was that people valuated items off of it. Saying that it affects that many people is a huge understatement when a single-digit number of cosmetics (that are supposed to be _hats_) were affected by this fix.

**Comment by MandM99** [comment]
> > The people attempting to get these bugs fixed are mostly doing this to laugh at people losing their item investments. It damages the item economy and will hurt not only the community, but the game's income.
> > @kisak-valve, we ask you to reconsider the fix since hundreds of people, including ones who are buying items off the SCM, had their inventory values demolished in the process after the update yesterday.
> 
> The only issue with the quickswitch UI getting fixed was that people valuated items off of it. Saying that it affects that many people is a huge overstatement when a single-digit number of cosmetics (that are supposed to be _hats_) were affected by this fix.

How can you say this when there's over 1100 Unusual Blast Defenses and approximately 800 combined Herald's Helms and Gauzed Gazes (all items that used the whole head regions and are a part of the +quickswitch menu bug) in existence, with all of them being priced due to the bug itself? Every single person who owns one just had its value drop by a solid amount, and many of the more expensive ones are now worthless.

https://backpack.tf/overview/Blast%20Defense

Many traders bought these items at their perceived value due to the glitch, including people who paid money for them on SCM. Its the equivalent of, say, Valve changing a valuable Counter Strike: Global Offensive AWP or AK-47 skin onto a lesser used weapon and calling it a day.

A 10 year old economy determining bug should not be fixed.  An alternative solution or form of compensation should be given to those who now own items worth 2 to over 25 times less than what they are now, at the bare minimum.

**Comment by rabscootle** [comment]
> > > The people attempting to get these bugs fixed are mostly doing this to laugh at people losing their item investments. It damages the item economy and will hurt not only the community, but the game's income.
> > > @kisak-valve, we ask you to reconsider the fix since hundreds of people, including ones who are buying items off the SCM, had their inventory values demolished in the process after the update yesterday.
> > 
> > 
> > The only issue with the quickswitch UI getting fixed was that people valuated items off of it. Saying that it affects that many people is a huge overstatement when a single-digit number of cosmetics (that are supposed to be _hats_) were affected by this fix.
> 
> How can you say this when there's over 1100 Unusual Blast Defenses and approximately 800 combined Herald's Helms and Gauzed Gazes (all items that used the whole head regions and are a part of the +quickswitch menu bug) in existence, with all of them being priced due to the bug itself? Every single person who owns one just had its value drop by a solid amount, and many of the more expensive ones are now worthless in comparison to what they once were.
> 
> https://backpack.tf/overview/Blast%20Defense
> 
> Many traders bought these items at their perceived value due to the glitch, including people who paid money for them on SCM. Its the equivalent of, say, Valve removing a valuable Counter Strike: Global Offensive AWP or AK-47 skin and placing it onto a lesser used weapon, then calling it a day without acknowledging the fact that people used the item valued at a certain amount in trades / on the SCM.
> 
> A 10 year old economy determining bug should not be fixed. An alternative solution or form of compensation should be given to those who now own items worth 2 to over 25 times less than what they are now, at the bare minimum.

The issue with this is that there was always precedent that the affected items were intended to conflict:
![image](https://user-images.githubusercontent.com/108029512/192659748-c70f490b-cd17-4b8e-a57a-446113e022b9.png)

Majority of the cosmetic owners probably never knew of this behavior and used them as intended (as a hat) because this is a (unintentional) bug caused by an obscure piece of UI. Investing on a bug has its risks and it's not the first time something of the sort happens.

**Comment by TaylorLeLievre** [comment]
> > Replying to [#4300 (comment)](https://github.com/ValveSoftware/Source-1-Games/issues/4300#issuecomment-1260106530)
> 
> The issue with this is that there was always precedent that the affected items were intended to conflict with normal hats: ![image](https://user-images.githubusercontent.com/108029512/192659748-c70f490b-cd17-4b8e-a57a-446113e022b9.png)
> 
> Majority of the cosmetic owners probably never knew of this behavior and used them as intended (as a hat) because this is a (unintentional) bug caused by an obscure piece of UI. Investing on a bug has its risks and it's not the first time something of the sort happens.

The thing that a lot of people don't realize about this situation is that Valve had tacitly approved of the existence of quickswitching in a previous patch. There used to be a bug caused by quickswitching 2 hats together where other people on the server would have their items unequipped, and bots started abusing it. Valve fixed that glitch without touching the actual quickswitch glitch despite it being the root of the problem. It was no coincidence that it was only after this update that quickswitch miscs began to significantly go up in price. The Wandering Wraith is another example, as on release the hype about it being a quickswitch misc prompted the creator to contact Valve about the item and change its equip region/do something about the bug. Yet Valve kept the region the same and keep the Wraith unboxable in unusual, and the bug had not been fixed until the original thread was created. I believe these bug fixes are being done by contractor independent of previous decisions, which is why this may only be happening now.

To respond to the point of "this isn't the first time something like this has happened", it really is. There are plenty of glitched items in TF2 with significant value that have gone untouched for years, and no glitched item has ever had its glitched qualities reverted before. I understand this isn't exactly the same as something like Strange Uniques/Strange Normal items existing, but there is no precedent for making major economic decisions like this. In fact, the situation with the Wandering Wraith set the opposite example.

At the end of the day, this is a bug that harms absolutely nobody, and there is only negative consequences if it is fixed. It would even be bad for Valve, as quickswitches are one of the biggest chase items whenever new effects come out and removing them would 100% lead to less unboxing.

**Comment by TF2CutContentWiki** [comment]
A bug is a bug, of course of course, and while Mr. Ed is a horse of course, all bugs must be fixed.

**Comment by tlelievre26** [comment]
> A bug is a bug, of course of course, and while Mr. Ed is a horse of course, all bugs must be fixed.

"Fix the bug because it is a bug" simply doesn't hold up. There's endless examples of bugs/exploits that turned out to be positive features for games, including but not limited to rocket jumping and the Spy being able to disguise as enemies. Especially when fixing a bug such as this strictly hurts average players, traders (big and small), and Valve themselves, there is literally no reason to fix this.

**Comment by TF2CutContentWiki** [comment]
The solution is to fix the equip regions, and patch these exploits. Instead of complaining that someone is trying to get an exploit fixed because it hurts your net profit (cause let's be honest, that's what this pushback is all about), go through the items_game.txt file, look at the equip regions, learn them and what each one conflicts with, and figure out what needs to be changed. Loads of cosmetics need equip region tweaks. That's the correct way to resolve this. Not trying to keep a silly exploit, which is hard to trigger and also gets reset by touching a locker among many other things (bet you didn't know that about this exploit eh?), from being patched.

**Comment by tlelievr27** [comment]
> > > A bug is a bug, of course of course, and while Mr. Ed is a horse of course, all bugs must be fixed.
> > 
> > 
> > "Fix the bug because it is a bug" simply doesn't hold up. There's endless examples of bugs/exploits that turned out to be positive features for games, including but not limited to rocket jumping and the Spy being able to disguise as enemies. Especially when fixing a bug such as this strictly hurts average players, traders (big and small), and Valve themselves, there is literally no reason to fix this.
> 
> The solution is to fix the equip regions, and patch these exploits. Instead of complaining that someone is trying to get an exploit fixed because it hurts your net profit (cause let's be honest, that's what this pushback is all about), go through the items_game.txt file, look at the equip regions, learn them and what each one conflicts with, and figure out what needs to be changed. Loads of cosmetics need equip region tweaks. That's the correct way to resolve this. Not trying to keep a silly exploit, which is hard to trigger and also gets reset by touching a locker among many other things (bet you didn't know that about this exploit eh?), from being patched.
> 
> There is plenty reason to fix this. You just think this busted exploit that isn't even guaranteed to work 9 times out of 10 is the 2nd coming of the quickswitch exploit and you want it kept. It's a gameplay logic bug that needs to be fixed.
> 
> OH and trying to play the "it hurts Valve's profit too" card cracks me up. That isn't gonna scare anyone into not fixing it.

Hello from a fresh account after [you blocked me from replying to this thread on my main](https://imgur.com/a/XG2VtPp). No better way to prove how well your argument holds up than silencing people who disagree.

Anyways I'm going to do a final address to each of the points you made.
1. The debate around this change has never been about items having janky equip regions. It's been about the fact that being able to wear these items as miscs was a major source of their value in trading, in addition to it being a neat and harmless trick that let you make some funny sets. If some hats like the Gauzed Gaze were moved to be miscs, sure then maybe we could do without the ability to glitch these hats together. But if a change like that were to happen it should be concurrent with the removal of this bug or come first.

2. You act like a large amount of people losing significant value in their items is not a good reason to oppose this change. In a game with an economy where items have proven real world value, needlessly tanking the value of certain items is absolutely a reason to not fix a bug, especially when there are no other benefits to doing so. I don't want to make this about the person on the other side of the argument, but it really does feel like you're doing this out of some odd sense of spite towards traders. [Your twitter](https://imgur.com/a/jGtgj2i) certainly doesn't do that notion any favors. Even then, theres well over 2000 unusual quickswitch miscs in existence and plenty of them are owned by people who bought them to make sets they liked. Its not just profit traders who are losing money on this.

3. The glitch is harder to do, but it is still possible to keep the items permanently equipped if they are whole-head cosmetics, just like with the old glitch.

4. Despite your claims that "there are plenty of reasons to fix this" and "this needs to be fixed", you've yet to actually list a single reason this glitch needs be fixed. Please tell me, how would the game tangibly benefit from the removal of this bug? I've already given a good list of ways this negatively impacts the game and the community. I guess its not as important to mention those kinds of things when you can try to stop others from responding.

6. Finally, don't put words in my mouth. Mentioning that Valve wouldn't make as much from unboxing is not a scare tactic, the drop in earnings would barely rustle a hair on Gaben's beard. Its just another reason why fixing this bug is an objectively bad idea with a negative outcome for everybody involved.

I'm going to assume this account will not be able to respond after this post, but I think I've said all I've had to say.

TLDR; I wholeheartedly believe this change offers a significant list of downsides and no tangible benefit to TF2, and I implore the issue creator to give me an example of any real benefit to making this change. As it stands @kisak-valve please do not fix this bug for the reasons mentioned above. 
And remember that blocking the people who disagree with you on a public discussion thread really isn't a good look.

**Comment by Concerned-Friend** [comment]
Hello there, I am writing because I am concerned about my friend. About 1 year ago he was going through major depression. Covid-19 took the life of his mother and a few months later he also lost his only furry friend and companion after 15 years together. Because of lockdowns TF2 was one of his only ways he connected with other human beings. Needless to say TF2 means a lot to him. Despite playing through the thick of the bot crisis, my friend stood tall against his depression, leaning heavily on community servers like Skial or Wonderland, where he made a few friends. Things seemed to be better for a while. Last night after I spoke to him he was not doing good. Even though it's not a lot of money, he invested a bit of his savings on some of these quickswitch miscs. After the latest update, my friend has been harassed and often gotten into heated arguments with other people about why this shouldn't have been dealt with the way it has been. He feels as if the one place that brought him joy did not care about him. Since this patch he's not been able to sleep, wear his funny bird heavy loadout, and his medic unusual combo he made. He's been very anxious and even started questioning his own existence and whether trying to continue or not. For my friend it's not been an issue of profit, or what one of the reasons many people seem to be against. In fact he only cares about expressing his creativity through funny and cool loadouts. TF2 and his love of hats has been a way for him to survive. Funny and cool loadouts he paid with his own savings from working as a frontline worker. Please Valve, if you care about your community, don't punish them for having spent money to enjoy the game you've allowed to exist as is for so long. If you are not giving us major updates, consider giving your player base quality of life improvements, community-driven updates, and not punitive updates that have no real benefit. I hope you rethink how you go forward.

Some suggestions:
-Make these items untradeable so people can continue enjoying what they paid for but not make profit
-Fix them so they are able to be used in combination of other hats. Obviously on a case-by-case basis depending on the hat. You can make them miscs
-Offer a refund for items that can no longer be quickswitched or offer users a buyout or replacement item in game
-Restore the ability to quickswitch, but disable the ability to unbox any new ones in unusual form
-Consider re-enabling unusual miscs to be unboxed (this is just for the future)

Thank you for your time in reading this and I sincerely wish you the strength and courage to do what's best for the community.
Charles AKA "Pita Pan Numba Wan Fan" on behalf of my friend

**Comment by TF2CutContentWiki** [comment]
> Hello there, I am writing because I am concerned about my friend. About 1 year ago he was going through major depression. Covid-19 took the life of his mother and a few months later he also lost his only furry friend and companion after 15 years together. Because of lockdowns TF2 was one of his only ways he connected with other human beings. Needless to say TF2 means a lot to him. Despite playing through the thick of the bot crisis, my friend stood tall against his depression, leaning heavily on community servers like Skial or Wonderland, where he made a few friends. Things seemed to be better for a while. Last night after I spoke to him he was not doing good. Even though it's not a lot of money, he invested a bit of his savings on some of these quickswitch miscs. After the latest update, my friend has been harassed and often gotten into heated arguments with other people about why this shouldn't have been dealt with the way it has been. He feels as if the one place that brought him joy did not care about him. Since this patch he's not been able to sleep, wear his funny bird heavy loadout, and his medic unusual combo he made. He's been very anxious and even started questioning his own existence and whether trying to continue or not. For my friend it's not been an issue of profit, or what one of the reasons many people seem to be against. In fact he only cares about expressing his creativity through funny and cool loadouts. TF2 and his love of hats has been a way for him to survive. Funny and cool loadouts he paid with his own savings from working as a frontline worker. Please Valve, if you care about your community, don't punish them for having spent money to enjoy the game you've allowed to exist as is for so long. If you are not giving us major updates, consider giving your player base quality of life improvements, community-driven updates, and not punitive updates that have no real benefit. I hope you rethink how you go forward.
> 
> Some suggestions: -Make these items untradeable so people can continue enjoying what they paid for but not make profit -Fix them so they are able to be used in combination of other hats. Obviously on a case-by-case basis depending on the hat. You can make them miscs -Offer a refund for items that can no longer be quickswitched or offer users a buyout or replacement item in game -Restore the ability to quickswitch, but disable the ability to unbox any new ones in unusual form -Consider re-enabling unusual miscs to be unboxed (this is just for the future)
> 
> Thank you for your time in reading this and I sincerely wish you the strength and courage to do what's best for the community. Charles AKA "Pita Pan Numba Wan Fan" on behalf of my friend

So this is the new tactic? Try to tug on their heartstrings? Come on dude. You're a new account created an hour ago specifically for this reason.

**Comment by GDGYGO** [comment]
If you invested a truckload of money on a hat whose value was entirely reliant on an unintended exploit that was also patched in the past (idk if anyone here remembers the stacking miscs quickswitch glitch), and you just... _hoped_ that _maybe_ it wouldn't get patched... it's on you. It's really that simple.

**Comment by TF2CutContentWiki** [comment]
Oh and lest I forget to bring this up again:

This remaining quickswitch exploit seen in that video I linked to in the report is very hard to get working. And at that, it can be undone quite easily/accidentally too. It's barely worth expending the time into trying to get it to work.

It's barely worth expending the time and effort to create multiple Github accounts and coming up with these Oscar-worthy sob stories about made-up friends to try to sway Valve into not fixing a broken barely working exploit.

Not to mention I'm pretty sure you can still momentarily disconnect your internet, equip two conflicting cosmetics, then reconnect and that would let you achieve the same end result.

**Comment by Moder112** [comment]
> As for my Twitter, what I post on it is neither here nor there. I can have an opinion and also report exploits. 

Except your opinions seem to affect your reports greatly as shown in #4069 where you advocated for fixing completly harmless items.

And when faced with criticism, you responded with nonsense memes and a rant about how much of a deal you are

>You don't know me very well do you? Does the name 404UNF/404 User Not Found ring a bell? Guy who co-founded TF2Classic? 😉👉👉 I'm already pretty much hated by most of the community for varying reasons, both legitimate and illegitimate, and I couldn't care less to be quite honest.

Seems like this is some weird powerplay for you.
I especially like how you ignore all of the valid points of the second guy, and choose to attack his character instead.
Valve gave you an inch so you now act all high and mighty, christ you're unbearable.

**Comment by TF2CutContentWiki** [comment]
> > As for my Twitter, what I post on it is neither here nor there. I can have an opinion and also report exploits.
> 
> Except your opinions seem to affect your reports greatly as shown in #4069 where you advocated for fixing completly harmless items.
> 
> And when faced with criticism, you responded with nonsense memes and a rant about how much of a deal you are
> 
> > You don't know me very well do you? Does the name 404UNF/404 User Not Found ring a bell? Guy who co-founded TF2Classic? 😉👉👉 I'm already pretty much hated by most of the community for varying reasons, both legitimate and illegitimate, and I couldn't care less to be quite honest.

God I love it when people don't get sarcasm. And also don't get the fact that *I don't care if people hate me for trying to get shit fixed or not*.

> Seems like this is some weird powerplay for you. I especially like how you ignore all of the valid points of the second guy, and choose to attack his character instead. Valve gave you an inch so you now act all high and mighty, christ you're unbearable.

🥱

**Comment by kisak-valve** [comment]
Hello everybody, friendly reminder that I'm a moderator for Valve's issue trackers on Github and not a Team Fortress 2 developer myself. I have no personal influence over changes to the game.

@tlelievre26 / @tlelievr27, upon reviewing your claim that you've been blocked, I've confirmed that is true, however, it hasn't been done by myself or a Valve developer. My guess is that the account has been frozen pending evaluation by Github staff. Regardless, creating multiple accounts to bypass a block is against Github's terms of service and that is not acceptable.

I'd like to urge everyone to keep this issue report civilized. Please note `Do not insult, harass, or demean anyone.` in [this issue tracker's conduct guidelines](https://github.com/ValveSoftware/Source-1-Games#conduct). Attacking someone else's character is both off-topic and non-productive. Please do not force me to take more severe action.

**Comment by TF2CutContentWiki** [comment]
> @tlelievre26 / @tlelievr27, upon reviewing your claim that you've been blocked, I've confirmed that is true, however, it hasn't been done by myself or a Valve developer. My guess is that the account has been frozen pending evaluation by Github staff. Regardless, creating multiple accounts to bypass a block is against Github's terms of service and that is not acceptable.

They're referring to me blocking them, which results in them being unable to comment on issues I've submitted. I doubt that constitutes "evasion" of a site block though.

**Comment by pinocchio-pinochet** [comment]
I don't believe the disconnecting from the internet method works anymore, having tried it myself. That said, I have come around to agreeing that the best course of action is to overhaul the equip regions of all cosmetics. I will go through items_game.txt and put together a list of suggestions.

**Comment by CrustTF2** [comment]
Better solution:

> Fix the bug with +quickswitch
> Change some of the more prominent but sensical quickswitch miscs to be real miscs (Wandering Wraith, Gauzed Gaze, Frag Proof Fragger, Mask Of The Shaman)
> Make Miscs unboxable as Unusuals again but in Cases only (not crates)
> Keep OG Miscs (ones from Crates) unobtainable from the Unusual Unbox Pool to leave the market unaffected (change them to limited items)
